package.json

```diff
@@ -24,6 +24,7 @@
     "react-bootstrap": "^2.9.2",
     "react-dom": "^18.2.0",
     "react-hook-form": "^7.49.2",
+    "react-loader-spinner": "^6.1.6",
```

Comment:
Instead of having a third party package for getting a spinner, we can have a look at this website which provides loaders and spinner with their code, and then we can have that code in our application and customize it as needed.
https://cssloaders.github.io/

Reply:
thank you Eng Ahmed , i will collaborate with Amira to implement the same loaders and toasters